src/ocamlorg_package/lib/ocamlorg_package.mli

```diff
@@ -145,7 +145,7 @@ val get_packages_with_name : state -> Name.t -> t list option
 (** Get the list of packages with the given name. *)
 
 val get_package_versions : state -> Name.t -> Version.t list option
-(** Get the list of versions for a package name. *)
+(** Get the reversed list of versions for a package name. *)
```

Comment:
I think it would be helpful to state what the ordering is. I.e. from latest to newest, or newest to latest.